app/assets/javascripts/accordion-with-descriptions.js

```diff
@@ -210,14 +208,14 @@
         });
       }
 
-      function openStoredSections($section) {
+      function openStoredSections ($section) {
         toggleSection($section);
         toggleIcon($section);
         toggleState($section.parent().find('.subsection__button'));
         // setOpenCloseAllText();
```

Comment:
As per above can we uncomment or delete?

Reply:
Okay - I'll put that in a separate pull request.